### PR TITLE
Works now on HTML5.

### PR DESCRIPTION
**Pull request by Vascofr**

### CODE REVIEW

File: UserInterface/FileBrowse/source/PlayState.hx [hunk 1]
```diff
@@ -17,6 +17,11 @@ import flash.display.Loader;
 import flash.display.LoaderInfo;
 import flash.net.FileReference;
 import flash.net.FileFilter;
+#elseif html5
```
Comment (Geokureli) [inline UserInterface/FileBrowse/source/PlayState.hx:20]: this should just be `#if (flash || html5)` openfl and flash imports are interchangeable in openfl/flixel

see here:
https://github.com/openfl/openfl/blob/develop/src/openfl/net/FileReference.hx#L1433
and here:
https://github.com/openfl/openfl/blob/develop/lib/flash-externs/src/flash/net/FileReference.hx#L31

**Review by Geokureli — CHANGES_REQUESTED** [review]

File: UserInterface/FileBrowse/source/PlayState.hx (hunk 1 above)
Comment (Vascofr) [reply UserInterface/FileBrowse/source/PlayState.hx]: Thanks!

**Review by Geokureli — APPROVED** [review]

**Comment by Geokureli** [comment]
@Vascofr did you specifically choose master over dev, or was master the default choice? please make PRs to dev, in the  future

**Comment by Vascofr** [comment]
@Geokureli I'm not sure, but I suppose it was the default since I don't see why I'd choose master. Sorry for the mistake, I'll try to remember next time.

**Comment by Geokureli** [comment]
only asking to see if can do anything on my end to prevent it. thanks!

**Pull request merged**